test/CommonMUnitTests.hs

```diff
@@ -4,42 +4,17 @@ import Prelude ()
 
 import Game.LambdaHack.Core.Prelude
 
-import qualified Data.EnumMap.Strict as EM
-import           Test.Tasty
-import           Test.Tasty.HUnit
+import Test.Tasty
+import Test.Tasty.HUnit
 
-import           Game.LambdaHack.Client.CommonM
-import           Game.LambdaHack.Common.Area
-import           Game.LambdaHack.Common.Kind
-import           Game.LambdaHack.Common.Level
-import           Game.LambdaHack.Common.Perception
-import           Game.LambdaHack.Common.Point
-import           Game.LambdaHack.Common.State
-import           Game.LambdaHack.Common.Time
-import           Game.LambdaHack.Content.TileKind
-import qualified Game.LambdaHack.Core.Dice as Dice
+import Game.LambdaHack.Client.CommonM
+import Game.LambdaHack.Common.Kind(emptyCOps)
+import Game.LambdaHack.Common.Perception (emptyPer)
```

Comment:
This inconsistent spacing might be caused by stylish-haskell not working for GHC 9.2.1 (stylish-haskel auto-formats this, if you have it set up in your editor). As I remarked on Discord, it's not yet updated for GHC 9.2, so GHC 9.0.2 is more convenient for the time being.